vision_agent_tools/shared_types.py

```diff
@@ -64,8 +65,20 @@ class Polygon(BaseModel):
     points: list[Point]
 
 
-class BoundingBox(BaseModel):
-    x_min: float
-    y_min: float
-    x_max: float
-    y_max: float
+# [x_min, y_min, x_max, y_max] bounding box
+BoundingBox = Annotated[list[int | float], Len(min_length=4, max_length=4)]
```

Comment:
I'm not sure about converting to a list, since BoundingBox usually represents a tuple of 4 elements.

Reply:
@hrnn I'm using the same type as defined [here](https://github.com/landing-ai/vision-agent-tools/blob/main/vision_agent_tools/tools/text_to_object_detection.py#L16) but making it standard across some models and tools. I believe that most of the models outputs are given as a list of four elements and the Annotation definition above fits into what we expect this value to be.